### DISCUSSION TITLE
Mixed up Compliance images between marketplaces

### DISCUSSION THREAD

**Discussion by NoxArt** [amzn/selling-partner-api-models#4395] [Q&A]
Hi,
I just uploaded 3 image feeds that each contains a compliance image (image type PS0x) that are linking to an image that depends on the marketplace
Feed 1 is linking to DE.jpg (german) ... sent to Amazon.de
Feed 2 is linking to BE.jpg (belgium) ... sent to Amazon.com.be
Feed 3 is linking to SE.jpg (swedish) ... sent to Amazon.se

Now the end result is that Amazon.de contains BE and SE images ... wut? Am I doing something wrong and this is by design? Or is it a bug?
I double checked everything that feeds contain correct links, links contain correct images, feeds were uploaded to correct marketplaces

I created a support case, but maybe someone here will know too

**Comment by NoxArt** [comment]
We noticed another seller has italian images on german marketplace, so it looks like it's a bug

**Comment by weilinggu** [comment]
Hi @NoxArt,

If you are using image attribute, check this [FAQ](https://developer-docs.amazon.com/sp-api/docs/listings-apis-faq#why-was-my-image-update-applied-globally) about image update. 
_"The image attributes provided by the [Listings Items API v2021-08-01](https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference) operations map images to ASINs at a global level where the ASIN exists, even if you specify the marketplace_id selector. This behavior is consistent with other experiences, such as Add a Product within Seller Central."_
In addition, in seller central, there is a tool you can use to upload country specific images.

As for uploading compliance image, have you looked into the use of `compliance_media`? Check this announcement [here](https://developer-docs.amazon.com/sp-api/changelog/developers-can-use-attributes-in-their-programmatic-listings-submissions-to-comply-with-gpsr)

Thanks
Weiling
Amazon Selling Partner Services

**Comment by NoxArt** [comment]
Thank you for reply. Does the same apply to feeds? And does it apply the same way to XML and JSON?

We cannot use a tool because we have like 40000+ listings

We use compliance_media, but on German marketplace they are being rejected without explanation ("reason" is empty). In recent weeks we have created multiple support cases and the cases are all "we are looking into this" without an answer for weeks - and the deadline is drawing near. We are unable to get anyone from support to help up and we fear Amazon will just suspend our whole portfolio on 13th

**Reply by weilinggu** [reply]
Yes, applies to feeds too. Yes, XML and JSON.
"on German marketplace they are being rejected without explanation ("reason" is empty)" - This is for submission using Listings API? What is the file format you submitted to compliance_media? 
compliance_media accepts these types: _Valid types include patient_fact_sheet, provider_fact_sheet, installation_manual, instructions_for_use, user_guide, user_manual, safety_information_ (according to this [annoucement](https://developer-docs.amazon.com/sp-api/changelog/developers-can-use-attributes-in-their-programmatic-listings-submissions-to-comply-with-gpsr))
Can you share your case IDs?

Thanks
Weiling
Amazon Selling Partner Services

**Comment by ritikacd** [comment]
Hi @NoxArt

Product safety images are applied locally, so if you update an image, it will show up in all the marketplaces where you have that ASIN. However, for warning and safety pdf documents, the language in the document must match the language of that store.
For DE, German must be used. Multilingual documents are allowed, but the content_language selected must align with the store/marketplace default language.

Currently, we don't support uploading warning and safety(compliance_media) documents through XSDs/customer templates. We are implementing that functionality and it will be available soon.

Please note to comply with Amazon's GPSR policy by December 13, 2024, please provide following attributes:

- EU Responsible Person
- Manufacturer contact
- Either of the 3:
  - Warning and safety PDFs (compliance_media) - only supported in JSON based APIs/Feed and should be a publicly available pdf
  - product safety & images using PS variants)
  - or acknowledgement that this isn't required using a safety attestation flag

Please reach out if you have any questions. 

Thanks,
Ritika
Amazon Selling Partner Services

**Reply by callain** [reply]
Hello

Are these attributes mandatory even if another seller has provided them? For me this is the point that is not clear.

If yes, that would mean in order to publish an offer a seller has to also provide partially some product info regarding safety and compliance.

Thanks

**Reply by ritikacd** [reply]
Hi @callain, 

If GPSR data is already present on an ASIN, you are not required to provide the data again. Additionally, if there are any issues, you'll see them when you try to list that ASIN by creating an offer against it. 

Thanks,
Ritika
Selling Partner API Services
